tasks/deploy/mainnet-config.ts

```diff
@@ -104,6 +104,7 @@ const addresses: ExtSystemConfig = {
         stablePool: "0x8df6EfEc5547e31B0eb7d1291B511FF8a2bf987c",
         bootstrappingPool: "0x751A0bC0e3f75b38e01Cf25bFCE7fF36DE1C87DE",
     },
+    balancerGaugeFactory: "0x4E7bBd911cf1EFa442BC1b2e9Ea01ffE785412EC",
```

Comment:
I think they just deployed a new version of this, should reach out and ask for the address